src/components/Comment/Comment.js

```diff
@@ -38,6 +43,12 @@ const styles = {
   })
 }
 
+export const CommentBody = ({children, style}) => (
```

Comment:
I think it would be better to have a comment root component in the mdast schema with the font sizes (serifRegular14/serifRegular16) and not expose this here—only have an wrapper with the profile pic margin here.